src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,43 @@
+.footer {
+  height: 56px;
```

Comment:
you already setting min-height with help of `makeStyles`. Am I missing something?

Reply:
No, forgot to delete it after created `Constants.js`